scenarios/waymo_motion/waymo_utility.py

```diff
@@ -0,0 +1,2202 @@
+            display_scenario_tags(tfrecord_tags, imported_tfrecord_tags)
+            tags, filter_display, stop_browser = prompt_filter_tags()
+            if stop_browser or filter_display == 5:
+                if filter_display:
+                    print_commands = True
+                continue
+
+            display_scenarios_in_tfrecord(
+                tfrecord,
+                scenario_dict,
+                tfrecord_tags,
+                imported_tfrecord_tags,
+                (tags, filter_display) if tags is not None else None,
+            )
+            time.sleep(1.2)
+            print_commands = True
+
+        elif re.compile("^export[\s]+(all|(?:\s*(\d+))+)", flags=re.IGNORECASE).match(
+            user_input
+        ):
+            input_lst = user_input.split()
+            if input_lst[1].lower() == "all":
+                valid_indexes = [i + 1 for i in range(len(scenario_ids))]
+            else:
+                valid_indexes = check_index_validity(
+                    input_lst[1:], len(scenario_ids), "export"
+                )
+                if len(valid_indexes) == 0:
+                    continue
+
+            # Prompt to allow filtering response with tags
+            display_scenario_tags(tfrecord_tags, imported_tfrecord_tags)
+            tags, filter_export, stop_browser = prompt_filter_tags()
+            if stop_browser:
+                return True
+            if filter_export == 5:
+                print_commands = True
+                continue
+
+            print(
+                "Enter the path to directory to which you want to export the scenarios:"
+            )
+            # Prompt to input target base path
+            target_base_path, stop_browser = prompt_target_path(default_target_path)
+            if stop_browser:
+                return True
+            if not target_base_path:
+                print_commands = True
+                continue
+
+            # Try exporting the scenario
+            exported = False
+            for idx in valid_indexes:
+                if tags is not None:
+                    if filter_scenario(
+                        tfrecord_tags.get(scenario_ids[idx - 1], []),
+                        imported_tfrecord_tags.get(scenario_ids[idx - 1], []),
+                        (tags, filter_export),
+                    ):
+                        pass
+                    else:
+                        continue
+                exported = (
+                    export_scenario(target_base_path, tfrecord, scenario_ids[idx - 1])
+                    or exported
+                )
+            if exported:
+                print(
+                    f"\nYou can build these scenarios exported using the command `scl scenario build-all {target_base_path}`"
+                )
+            else:
+                if tags:
+                    print("No scenarios were exported since no tags matched\n")
+            time.sleep(1)
```

Comment:
What is the purpose of this `sleep`? 

- Is there a thread or process running that requires it?